lessons/pydata/pandas_types/index.ipynb

```diff
@@ -0,0 +1,10723 @@
+{
```

Comment:
<p>Už jsme se nějak dohodli na společném repu nebo tak něčem? Případně máme to v plánu? Já jen ať nám tu nezůstane tvoje osobní repo napořád. Ne že bys ho mazal... ale tak ať to máme pohromadě.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
<p>Spíš jsme to nedořešili, tohle opravdu chápu jako dočasné.</p>

Reply:
<p><a href="https://github.com/PyDataCZ/naucse.python.cz/issues/40" target="_blank">https://github.com/PyDataCZ/naucse.python.cz/issues/40</a></p>